### ISSUE TITLE
Consistency of Fine-Tuning Results with 1 GPU vs. 8 GPUs for Action Recognition

### ISSUE THREAD

**Issue by beitong95** [zhaoyue-zephyrus/AVION#13]
Hello, thank you for maintaining this fantastic repository!

I’m currently fine-tuning on the EPIC-KITCHENS Action Recognition dataset but have access to only 1 GPU. I wanted to ask if there would be any noticeable performance difference in the results obtained from fine-tuning with a single GPU compared to training with 8 GPUs. 

Thank you for your guidance!

**Comment by zhaoyue-zephyrus** [comment]
Hi @beitong95 ,

For fine-tuning, I don't think there will be a big difference between the 1-GPU or 8-GPU settings even though I didn't try. You may want to linearly scale down the learning rate though.

Best,
Yue

**Comment by beitong95** [comment]
Hi Yue,

Thank you for your quick response! I will proceed with 1-GPU and will share the training log here. So far, I don't notice a significant difference.

I believe the learning rate in `main_lavila_finetune_cls.py` has been linearly scaled, as shown below:

```python
total_batch_size = args.batch_size * dist_utils.get_world_size()
args.lr = args.lr * total_batch_size / 128
args.lr_start = args.lr_start * total_batch_size / 128
args.lr_end = args.lr_end * total_batch_size / 128
```

Best,  
Beitong

**Comment by zhaoyue-zephyrus** [comment]
Oh yeah, you're right. I forgot that I had written that already :P

**Comment by beitong95** [comment]
[log_one_RTX4090.txt](https://github.com/user-attachments/files/17673136/log_one_RTX4090.txt)

Here is the log of training with one GPU on the EK100 action recognition task. The performance is very close 0.494 (1GPU) vs 0.495 (8GPU)

**Comment by zhaoyue-zephyrus** [comment]
Nice! Glad to know that fine-tuning can be reproduced with 1 GPU!
